lib/mapping.gi

```diff
@@ -165,17 +165,22 @@ InstallGlobalFunction( Image, function ( arg )
       map := arg[1];
       elm := arg[2];
 
-      if     FamSourceEqFamElm( FamilyObj( map ), FamilyObj( elm ) ) then
+      if not IsMapping( map ) then
+        ErrorNoReturn( "<map> must be a mapping" );
```

Comment:
But why? Before this, one could have done `Image(generalMap, set)` to return the set of (generalized) images for the generalized map `generalMap`. I think that was intentionally supported.

Reply:
I think you're right. I updated the PR. The next thing to do now is to write some tests.